python/sglang/bench_serving.py

```diff
@@ -1316,6 +1438,11 @@ def set_ulimit(target_soft_limit=65535):
         type=int,
         help="If not set, the default port is configured according to its default value for different LLM Inference Engines.",
     )
+    parser.add_argument(
+        "--grpc-port",
+        type=int,
+        help="If not set, the default port is configured according to its default value for different LLM Inference Engines.",
```

Comment:
Let's comment this is only for sglang backend and used only when grpc backend is used with --backend=="sglang-grpc"?